test/python/fe_api/grouped_gemm/test_grouped_gemm_dswiglu_jax.py

```
@@ -0,0 +1,104 @@
+
+import numpy as np
+import pytest
+
+jax = pytest.importorskip("jax")
+ml_dtypes = pytest.importorskip("ml_dtypes")
+import jax.numpy as jnp
+
+from fe_api.gemm.test_gemm_amax_jax import skip_unless_sm100
+
+
+def _make_jax_inputs(m=256, n=128, k=128, l=2):
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)